rfcs/2021-04-07-6330-emitting-multiple-log-events-from-remap-transform.md

```diff
@@ -0,0 +1,182 @@
+# RFC 6330 - 2021-04-07 - Emitting multiple log events from remap transform
+
+This RFC describes an approach for emitting multiple log events from a remap transform.
+
+[Credit to @JeanMertz](https://github.com/timberio/vector/pull/7038#discussion_r609476064) for this revised approach.
+
+## Scope
+
+In:
+
+- Turning a log event into multiple log events
+
+Out:
+
+- Turning a log event into multiple metrics
+- Turning a metric into multiple metrics
+- Turning a metric into multiple logs
+
+## Motivation
+
+Currently the remap transform can only modify an event as it is passing through. Users have asked [1][] [2][] to be able to transform an incoming event into multiple events. This is useful, for example, when parsing an incoming JSON payload that contains an array for which you'd like to publish an event for each element.
```

Comment:
```suggestion
Users have asked [1][] [2][] to be able to transform an incoming event into multiple events. This is useful, for example, when parsing an incoming JSON payload that contains an array for which you'd like to publish an event for each element.
```

I don't think it's relevant to mention the remap transform in this case, as it is complimentary, but serves a different purpose.